person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age, name: 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [ ] [Optional] [Highly Recommended] It would be better professionally if you created the `id` automatically, not passing it as a parameter as you did.

[Hint] You can make it like that:
```ruby
def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
```